jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/web.xml

```diff
@@ -151,6 +151,16 @@
         <url-pattern>/*</url-pattern>
     </filter-mapping>
 
+    <!--If request method to login page isn't GET or POST, filter changes it to GET-->
```

Comment:
By "why" I meant "please left a comment in source code to explain these lines" :) 

A better solution could be to map this or a different filter to `/api/*`